frontend/src/modules/contributor/components/edit/identity/contributor-identity-edit.vue

```diff
@@ -132,7 +126,12 @@ const platform = computed(() => CrowdIntegrations.getConfig(form.platform));
 const updateIdentity = () => {
   sending.value = true;
 
-  updateContributorIdentity(props.contributor.id, props.modelValue.id, form)
+  updateContributorIdentity(props.contributor.id, props.modelValue.id, {
+    ...form,
+    verified: false,
+    integrationId: null,
+    sourceId: null,
```

Comment:
For people:
For emails, can you also change platform to "custom"? Otherwise source will be GitHub, and that would be wrong since we changed it.
![Screenshot 2024-08-21 at 12 03 54](https://github.com/user-attachments/assets/2808f404-0dce-4b76-bc24-1292eee01ebe)

For organizations:
Same comment above but for domains.